app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.players.elves.elf import Elf
+from app.players.player import Player
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(team: list[Player]) -> int:
+    total = 0
+    for i in team:
+        total += i.get_rating()
+    return total
```

Comment:
Use `sum` and pass a generator expression there in this function as mentioned in the checklist.